lib/colorls/flags.rb

```diff
@@ -49,8 +53,19 @@ def add_sort_options(options)
       options.separator ''
       options.separator 'sorting options:'
       options.separator ''
-      options.on('--sd', '--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:sort] = :dirs }
-      options.on('--sf', '--sort-files', 'sort files first')                                   { @opts[:sort] = :files }
+      options.on('--sd', '--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:group] = :dirs }
+      options.on('--sf', '--sort-files', 'sort files first')                                  { @opts[:group] = :files }
+      options.on('--ss', '--sort-size', 'sort by size')                                       { @opts[:sort] = :size }
+      options.on('-st', 'sort by modification time, newest first')                            { @opts[:sort] = :time }
+      options.on('--sort=WORD', %w[none time size], 'sort by WORD instead of name: none, time (-st)') do |word|
```

Comment:
@avdv - Given that `--st` and `--ss` are defined above, is this block required?

Reply:
Actually, I think `-st` is really wrong as we have actually worked towards removing multi-char short options in the past and this introduces another one. Since `--sf` and `--ss` only exist due to keep backward compat / for historical reasons, we should not add `--st`, IMO.

`ls` uses `-t`. So `colorls` should do this too. I'll change that and open another PR...